tests/FakeItEasy.Tests.Approval/ApprovedApi/FakeItEasy/net462.verified.txt

```diff
@@ -80,6 +80,14 @@ namespace FakeItEasy
         public static string StartsWith(this FakeItEasy.IArgumentConstraintManager<string> manager, string value) { }
         public static string StartsWith(this FakeItEasy.IArgumentConstraintManager<string> manager, string value, System.StringComparison comparisonType) { }
     }
+    public abstract class ArgumentEqualityComparer<T> : FakeItEasy.IArgumentEqualityComparer
+    {
+        protected ArgumentEqualityComparer() { }
+        public virtual FakeItEasy.Priority Priority { get; }
+        public abstract bool AreEqual(T? expectedValue, T? argumentValue);
```

Comment:
`protected`? I think the build system will support me here.

Reply:
Ah, yes. Forgot to run the approval tests after I changed this.